clar2wasm/src/words/equal.rs

```diff
@@ -707,8 +707,13 @@ fn wasm_equal_tuple(
 
     // if this is a 1-tuple, we can just check for equality of element
     if depth == 1 {
-        let (ty, range) = wasm_ranges.next().unwrap();
-        let nth_ty = nth_types.next().unwrap();
+        let (ty, range) = wasm_ranges.next().ok_or_else(|| {
```

Comment:
It seems like the checks on `depth` protect us here, but I think it's fine to be extra cautious, to protect from future changes in this code.

Reply:
Understood. Am I supposed to leave it as it is?

Reply:
Yup, it's good as is. Was just thinking out loud. Sorry, should have been more clear.